contributing/development-setup.md

```diff
@@ -69,10 +69,9 @@ Use [*Conventional Commits*](https://www.conventionalcommits.org/) to name pull
 - `style: Description of the PR`: The commit updates or reformats the style of the source code, but does not otherwise change the product implementation (this correlates with Minor in Semantic Versioning).
```

Comment:
Was thinking if we should update our pull request template required check list with a point on the lines: If the changes include a breaking change, or a new feature that contributes to a coming major version (e.g. v1.15.0), a migration guide must be created or updated.

Reply:
good idea, PR template changed